### ISSUE TITLE
Tabview : Accessibility -  ARIA property 'aria-controls' value is not valid The ARIA property must reference a non-empty unique id of an existing element that is visible

### ISSUE THREAD

**Issue by ragul0205** [primefaces/primereact#8356]
### Accessibility Violation

The 'id' "pr_id_441_content" specified for the ARIA property 'aria-controls' value is not valid
The ARIA property must reference a non-empty unique id of an existing element that is visible

Why is this important?
Any element that uses an ARIA property must reference a valid id attribute. This enables assistive technologies to precisely identify the referenced element.

Element location
```
<a
    data-pc-section="headeraction"
    aria-disabled="false"
    aria-selected="false"
    aria-controls="pr_id_441_content"
    tabindex="-1"
    class="p-tabview-nav-link"
    role="tab"
    id="pr_id_44_header_1">
```
What to do
Update the ARIA property value with the unique id value of a valid visible element.

For example:
```
<form action="" method="post">
      <label for="ln1">Last Name</label>
      <input type="text" name="lastname" id="ln1" value="enter last name" title="enter last name" aria-describedby="ln2">
</form>
<div id="ln2" role="tooltip">Last Name must be alpha only characters</div>
```

Rule ID: aria_id_unique
Reason ID: Fail_1

<img width="1919" height="877" alt="Image" src="https://github.com/user-attachments/assets/4f1ef269-e3ee-4b48-a81f-52cccbccf114" />

### System Information

```Shell
Can be reproduced in every environment using IMB Accessibility checker
```

### Steps to reproduce the behavior

1. Create a `Tabview` with more than 1 `TabPanel`.
2. Run  IMB Accessibility checker
3. Check Violations

### Expected behavior

Every   `Tab link`  must have corresponding `Tab Content` available in DOM.

Example  below.

```
<div id="pr_id_441_content" role="tabpanel" aria-labelledby="pr_id_44_header_1" >
</div>
```

**Comment by Aryan3902** [comment]
The issue seems to be because of the tab headers and the tab content panels, the tab content component is using Lazy Loading. It has removed the inactive tab panels from the HTML.

In the above given screenshot and description -
Header 2 points to ID: pr_id_441_content. (you can see in the screenshot 442 id is also throwing violation which can point to Header 3)
The DOM contains only: pr_id_440_content (Panel 1). (Rest will be lazy loaded)

Result: The browser looks for the ID referenced by Header 2, cannot find it, and flags the aria_id_unique violation.
I believe in PrimeReact we can use the prop `renderActiveOnly` to disable lazy loading and hence prevent the violation.

What do you think @melloware

**Comment by melloware** [comment]
@Aryan3902 if you want to submit a PR feel free if you think you know the fix!
